app/controllers/api/students/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::Students::CommentsController < Api::BaseController
+    # TODO (warren) Check if we should extend ApiBaseController
+
+    respond_to :json
```

Comment:
Don't actually need `respond_to :json` since it'll inherit that from `Api::BaseController`.